text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,537 @@
+* Create StackSet artifact type in the Cloud Assembly
+* Create Bootstrapless StackSet Synthesizer
+* Update CLI to collect StackSet artifact types in addition to Stacks
+* Deployment code for creating/updating stack sets
+
+### Are there any open issues that need to be addressed later?
+
+* CDK asset handling
+* Future support for service managed stack sets
+* CDK Pipelines
+
+---
+# Implemenation Details
+
+# Requirements
+
+## In Scope Features
+
+* Create Self-Managed Stack Set
+  * This should create the stack set directly, as opposed to the [CfnStackSet](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cloudformation.CfnStackSet.html) construct which is defined in a single CloudFormation stack that deploys the stack set
+* Update Self-Managed Stack Set
+  * Update includes updating **all existing** stack set instances 
+  * Set Operation Preference Configuration
+
+## Out of Scope Features
+
+* Delete Self-Managed Stack Set
+* Service Managed Stack Set
+* Add/Remove stack set instances through CDK
+* Update to only specific stack set instances
+* Stack Set Instance Level Parameter Overrides
+* CDK Assets (CDK File Assets/CDK Docker Assets)
+  * More information on CDK asset handling and complexities described in the [CDK Assets Handling](#4-cdk-assets-handling) section.
+
+## Assumptions
+
+* Parent account contains Stack Set administration role and child accounts contains Stack Set execution role for enabling stack set operations
+  * https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs-self-managed.html
+  * The Stack Set administration role in the parent account is used by CloudFormation to assume the execution role in the child account. This role can be created manually or may be created through a CDK stack. This `role arn` is passed into the stack set CDK construct properties 
+  * The Stack Set execution role in the child is the role assumed by the administration role in the parent account, and utilized to provision resources in the child account. This role should be provisioned manually, and the `role name` is passed into the stack set CDK construct properties
+* Stack Set Instances
+  * In this initial release, stack set instances should be added manually to stack set after it was initially created by CDK
+  * Since this initial iteration does not include CDK assets support such as files/docker images, bootstrapping multiple regions is not required. Bootstrapping a single region in the parent account may be required for templates larger than 50kb. 
+
+## Non-Functional Requirements
+
+* Implementation does not incur breaking changes for existing users or future feature implementation
+  * I.e Current CDK users should not have to convert any other CDK app implementations to accommodate for changes implemented in this design
+
+* Implementation is extensible for future features such as:
+  * Self-managed Stack Sets
+  * Stack Set Instance Add/Remove
+  * Additional asset handling
+
+# Design
```

Comment:
I'm wondering whether this section is useful here. I feel like it goes into a lot of detail that we generally don't go into in the RFC.

@linsona do you want our feedback on some specific part of this section? Because if not, I actually think we should remove it, and this should be just a shared document that's used inside of your team.

Reply:
Removing the design section